<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_DadosConsultasV</class>
 <widget class="QWidget" name="frm_DadosConsultasV">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #008080;
    border-radius: 8px;
}</string>
  </property>
  <widget class="QLabel" name="lbl_nome">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 16px;
    color: #333333;
    font-weight: bold;
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.4);
}
</string>
   </property>
   <property name="text">
    <string>Nome/Produto:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_nome">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px solid #cccccc; 
    border-radius: 5px; 
    padding: 6px; 
    font-size: 14px; 
    background-color: #ffffff;
    transition: all 0.3s ease;
	color: #000000;
}

QLineEdit:hover {
    border: 2px solid #3f51b5; 
    background-color: #f5f5f5; 
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_qtd">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 16px;
    color: #333333;
    font-weight: bold;
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.4);
}
</string>
   </property>
   <property name="text">
    <string>Quantidade:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Valor">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 16px;
    color: #333333;
    font-weight: bold;
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.4);
}
</string>
   </property>
   <property name="text">
    <string>Valor:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancelar">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #ffebee; 
    border: 2px solid #ffcdd2;
    border-radius: 10px;
    color: #b71c1c; 
    font-size: 14px;
    font-weight: bold;
    padding: 10px 16px;
    background-image:url(:/icon_cancelar/cancelar.png);
    background-repeat: no-repeat;
    background-position: center;
    padding-left: 40px;
    transition: all 0.3s ease;
}

QPushButton:hover {
    background-color: #ffcdd2;
    border-color: #e57373;
    color: #d32f2f; 
}

QPushButton:pressed {
    background-color: #e57373;
    border-color: #b71c1c;
    padding-left: 44px;
    padding-top: 2px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_qtd">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px solid #cccccc; 
    border-radius: 5px; 
    padding: 6px; 
    font-size: 14px; 
    background-color: #ffffff;
    transition: all 0.3s ease;
	color: #000000;
}

QLineEdit:hover {
    border: 2px solid #3f51b5; 
    background-color: #f5f5f5; 
}
</string>
   </property>
   <property name="inputMask">
    <string>00000000000000000000000000000000000000000000000000</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_valor">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px solid #cccccc; 
    border-radius: 5px; 
    padding: 6px; 
    font-size: 14px; 
    background-color: #ffffff;
    transition: all 0.3s ease;
	color: #000000;
}

QLineEdit:hover {
    border: 2px solid #3f51b5; 
    background-color: #f5f5f5; 
}
</string>
   </property>
   <property name="inputMask">
    <string>R$</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_nome_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 2px solid #cccccc; 
    border-radius: 5px; 
    padding: 6px; 
    font-size: 14px; 
    background-color: #ffffff;
    transition: all 0.3s ease;
	color: #000000;
}

QLineEdit:hover {
    border: 2px solid #3f51b5; 
    background-color: #f5f5f5; 
}
</string>
   </property>
   <property name="inputMask">
    <string>00/00/0000</string>
   </property>
   <property name="text">
    <string>//</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_nome_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 16px;
    color: #333333;
    font-weight: bold;
    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.4);
}
</string>
   </property>
   <property name="text">
    <string>Data:</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icon_cadastrar.qrc"/>
  <include location="icon_cancelar.qrc"/>
 </resources>
 <connections/>
</ui>
